src/popupwin.c

```diff
@@ -2844,6 +2844,51 @@ f_popup_settext(typval_T *argvars, typval_T *rettv UNUSED)
     popup_adjust_position(wp);
 }
 
+/*
+ * popup_setbuf({id}, {bufnr})
+ */
+    void
+f_popup_setbuf(typval_T *argvars, typval_T *rettv UNUSED)
+{
+    int		id;
+    win_T	*wp;
+    buf_T	*buf;
+
+    if (in_vim9script()
```

Comment:
As this is a new function, this check for the Vim9 script can be removed.  In the existing functions this check was added to not break backward compatibility with legacy scripts.

Reply:
Thanks, adjusted.